Developer will

ZefengQiu opened the pull request:
add something

Commit:
add loading cell and singleton model to get the parsed html data

Commit:
show the parsing result in the list view controller

Commit:
again

honghaoz commented on UW Info Session 2.0/UW Info Session/UW Info Session/Network/InfoSessionSourceHTMLParser.swift:
```diff
@@ -51,31 +51,39 @@ struct InfoSessionSourceHTMLParser {
     
     private func processTrSessionGroupToDict(trSession: [JiNode]) -> [[String: String]] {
-        var result = [[String: String]]()
+        var resultunit = [[String: String]]()
```
`var resultUnit` will be better

Commit:
detail view controller UI updating

Commit:
create scroll view controller to show the map of UW;  improve detail UI a little; create a changeable line note cell which the height of that cell can adjust how many line of the note.
But there still problem wait to be solved, when the keyboard show up, it block the table view content , cannot see the note cell.

Commit:
add WebKit for the info Session website cell, but there is a bug when back to the detail view controller for the Web page view controller, said : terminating with uncaught exception of type NSException. still need to be fixed

Commit:
Improved detail view UI, including:  scroll the table to show noteCell when keyboard is up, add alert Cell to response the reminder Cell switch.
Considering adding today extension.

Commit:
add today extension for info session

Commit:
update for Swift 2, and add google map

Commit:
small error corrected, but still have two warming concerning the core data

Commit:
add favourite view

Commit:
add tab view, but with a problem

honghaoz committed:
Fixed tabor selections

Commit:
Merge branch 'master' of https://github.com/honghaoz/UW-Info into developer_Will

Commit:
no message

Commit:
Merge branch 'developer_Will' of https://github.com/honghaoz/UW-Info into developer_Will

# By Honghao Zhang
# Via Honghao Zhang
* 'developer_Will' of https://github.com/honghaoz/UW-Info:
  Fixed tabor selections

Conflicts:
	UW Info Session 2.0/UW Info Session/UW Info Session/Component/RootViewController.swift
